ui/apps/platform/src/Containers/VulnMgmt/List/Images/VulnMgmtListImages.js

```diff
@@ -19,7 +19,10 @@ import { LIST_PAGE_SIZE } from 'constants/workflowPages.constants';
 import WorkflowListPage from 'Containers/Workflow/WorkflowListPage';
 import workflowStateContext from 'Containers/workflowStateContext';
 import { imageWatchStatuses } from 'Containers/VulnMgmt/VulnMgmt.constants';
-import { IMAGE_LIST_FRAGMENT } from 'Containers/VulnMgmt/VulnMgmt.fragments';
+import {
+    IMAGE_LIST_FRAGMENT,
+    OLD_IMAGE_LIST_FRAGMENT,
```

Comment:
I think in other instances you've been appending a `_UPDATED` term at the end of the fragment query. Should we do the same thing here instead of prepending `_OLD`?

Reply:
If anything, I would change the places that use UPDATED to be plain, and their counterparts to be OLD, but it is definitely short-lived code (even if short lived means until the end of the year.)